src/javascript/SelectorTypes/Picker/Picker2.jsx

```diff
@@ -138,7 +138,7 @@ export const Picker2 = ({field, value, editorContext, inputContext, onChange, on
                     editorContext={editorContext}
                     pickerConfig={pickerConfig}
                     initialSelectedItem={fieldData && fieldData.map(f => f.path)}
-                    accordionItemProps={parsedOptions.accordionItem}
+                    accordionItemProps={parsedOptions.accordionItem === undefined ? pickerConfig.accordionItem : parsedOptions.accordionItem}
```

Comment:
I'm just not sure if the behaviour would be ok if you have field override + accordion config in picker config ? I would expect override to just extend the normal config, not replace it